Formula/p/pakchois.rb

```diff
@@ -30,8 +30,25 @@ def install
     # Fix flat namespace usage
     inreplace "configure", "${wl}-flat_namespace ${wl}-undefined ${wl}suppress", "${wl}-undefined ${wl}dynamic_lookup"
 
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
+    args = []
+    # Help old config scripts identify arm64 linux
+    args << "--build=aarch64-unknown-linux-gnu" if OS.linux? && Hardware::CPU.arm? && Hardware::CPU.is_64_bit?
+
+    system "./configure", *args, *std_configure_args
     system "make", "install"
   end
+
+  test do
+    (testpath/"test.c").write <<~C
+      #include <pakchois/pakchois.h>
+      #include <stdio.h>
+
+      int main(void) {
+          return 0;
+      }
```

Comment:
We should try to test at least one function in `pakchois.h` here.

Reply:
make sense 👍